build/recipes.mk

```diff
@@ -15,7 +15,6 @@
 # ------------------------------------------------------------
 
 BICEP_RECIPE_TAG_VERSION?=latest
-RAD_BICEP_PATH?=$${HOME}/.rad/bin
```

Comment:
This is not being used anywhere in the code. I did a search in the whole project. Please correct me if I am wrong.

Reply:
If it's unused then it's good for us to delete !